common/json_compare.go

```diff
@@ -0,0 +1,111 @@
+package common
+
+// Compare the JSON values (intended to compare override json and config output
+// in case of SLZ but can be used anywhere)
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sort"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+)
+
```

Comment:
All of these functions should have a unit test. If we call this file `json_helpers.go` then the unit tests would go in `json_helpers_test.go`.

Reply:
Noted. Created the test file with name `json_helpers_test.go`